components/automate-ui/src/app/entities/layout/layout-sidebar.service.ts

```diff
@@ -243,15 +243,6 @@ export class LayoutSidebarService implements OnInit, OnDestroy {
       this.sidebars = this.updateVisibleValues(sidebars);
     }
 
-    ngOnInit() {
-      this.updateSidebars();
-    }
```

Comment:
Bonus to this PR: not related to the bug being addressed, but for the same reasons this code would never be executed, so applied the same fix.